pom.xml

```diff
@@ -31,9 +32,11 @@
   </scm>
 
   <properties>
-    <revision>1.14</revision>
+    <revision>1.15</revision>
+    <changelist>-SNAPSHOT</changelist>
```

Comment:
Looks like these properties are dupes.

Reply:
You are correct, this was something that slipped through when merging